src/modules/nftCollections/tests/collections-transactions.service.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { IssueCollectionRequest, SetNftRolesRequest, StopNftCreateRequest, TransferNftCreateRoleRequest } from '../models/requests';
+import { CollectionsTransactionsService } from '../collections-transactions.service';
+
+describe('Collections Transactions Service', () => {
+  let service: CollectionsTransactionsService;
+  let module: TestingModule;
+  const ownerAddress = 'erd1dc3yzxxeq69wvf583gw0h67td226gu2ahpk3k50qdgzzym8npltq7ndgha';
+
+  beforeEach(async () => {
+    module = await Test.createTestingModule({
+      providers: [CollectionsTransactionsService],
+    }).compile();
+
+    service = module.get<CollectionsTransactionsService>(CollectionsTransactionsService);
+  });
+
+  it('should be defined', () => {
+    expect(service).toBeDefined();
+  });
+
+  describe('issueToken', () => {
+    it('returns build transaction for issue non fungile with right arguments', async () => {
```

Comment:
typo in `fungile`. as discussed will be fixed in next pr

Reply:
done